content/docs/command-reference/add.md

```diff
@@ -153,10 +153,12 @@ not.
   > Additionally, this typically requires an external cache setup (see link
   > above).
 
-- `-o <path>`, `--out <path>` - destination `path` to make a local target copy,
-  or to [transfer](#example-transfer-to-cache) an external target into the cache
-  (and link to workspace). Note that this can be combined with `--to-remote` to
-  avoid storing the data locally, while still adding it to the project.
+- `-o <path>`, `--out <path>` - destination `path` inside the workspace to link
+  (or copy) a data target, which will now be tracked by DVC. Note that combining
+  this with an
+  [external cache transfer](#example-transfer-to-an-external-cache), or with the
+  `--to-remote` option, let's you avoid storing an external target locally,
+  while still adding it to the project.
 
 - `--to-remote` - import an external target, but don't move it into the
   workspace, nor cache it. [Transfer it](#example-transfer-to-remote-storage) it
```

Comment:
transfer probably is not the right term (we don't remove it, right?)

Reply:
> transfer probably is not the right term

Yeah I was thinking that too earlier 🤔 but we already use that term a lot and it's kind of nice. Let me think ⌛

Reply:
OK I removed the term in favor of more explicit phrases like "store a copy".

Reply:
\* the term "transfer"